building/github/contributors-pull-request-guide.md

```diff
@@ -1,41 +1,40 @@
 # The Contributors' Guide to Pull Requests
 
 Thank you for wanting to contribute to Exercism!
-Before creating a pull request, please read the [how to make a great Pull Request][how-to-make-a-great-pr] guide which aims to help you with the non-technical side of pull requests so that you have a great experience and help our maintaining team at the same time.
+Before creating a pull request, please read the [how to make a great pull request][how-to-make-a-great-pr] guide which aims to help you with the non-technical side of pull requests so that you have a great experience and help our maintaining team at the same time.
 
 This documents contains some additional, Exercism-specific guidelines for different types of pull requests.
 
 ## Exercise Pull Requests
 
 Creating a pull request for a Concept Exercise or Practice Exercise can be daunting with the many rules around these types of exercises.
-For this reason, it can take a maintainer 2 to 3 hours to review and can result in dozens of comments.
-
+For this reason, it can take a maintainer two to three hours to review and can result in dozens of comments.  
 To help you, there'll be one primary reviewer commenting on your pull request.
 
 Don't be discouraged by the number of review comments; it is perfectly normal for an exercise to go through several rewrites before arriving at something both you and the primary reviewer are happy with.
 
 Here are some recommendations to help streamline the process even more.
```

Comment:
With the "Recommendation: " prefix removed, maybe we can add a recommendations header instead?

```suggestion
## Recommendations

Here are some recommendations to help streamline the process even more.
```